src/model/geometrical_models.py

```diff
@@ -1,4 +1,4 @@
-from epipeODE import Cell
+from embryo import Cell
```

Comment:
```suggestion
from embryo import Cell
I did not see the change to epipeODE before the next pull request
```
from epipeODE import Cell